packages/frontend/src/creation-form/utils/outcomes-configuration.ts

```diff
@@ -26,6 +27,22 @@ export const outcomeConfigurationFromOracleData = (
                   }
                 : DEFAULT_OUTCOME_CONFIGURATION;
         }
+        // defillama oracle template
+        case 2: {
+            return {
+                automaticallyFilled: false,
```

Comment:
Should we spread `DEFAULT_OUTCOME_CONFIGURATION` and  then set `binaryTogglable` to false here? I feel like it would be more future proof no?